setup.py

```diff
@@ -90,7 +90,7 @@
         'websocket-client',
         'yaspin',
         # KFP runtime dependencies
-        'kfp-notebook>=0.22.0',
+        'kfp-notebook>=0.23.0',
```

Comment:
Should we be perhaps a bit more conservative?
```suggestion
        'kfp-notebook~=0.23.0',
```

Reply:
Yes, I thought I had changed that.